src/api/app/views/webui2/webui/project/show.html.haml

```diff
@@ -24,10 +24,24 @@
     = render partial: 'webui2/shared/buildresult_box', locals: { project: @project.name }
   .list
     .card
-      %h5.card-header
-        Packages
-        = "(#{@packages.length})"
-      = render partial: 'project_packages'
+      .bg-light
+        %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap{ role: 'tablist', id: 'packages-tabs' }
+          %li.nav-item
+            %a.nav-link.active{ id: 'packages-tab', href: '#packages', role: 'tab', data: { toggle: 'tab' },
+            aria: { controls: 'packages', selected: 'true' } }
+              Packages
+              = "(#{@packages.length})"
+          - if @ipackages.present?
+            %li.nav-item
+              %a.nav-link{ id: 'inherited-packages-tab', href: '#inherited-packages', role: 'tab', data: { toggle: 'tab' },
+              aria: { controls: 'inherited-packages', selected: 'false' } }
+                Inherited Packages
+                = "(#{@ipackages.length})"
+      .tab-content{ id: 'packages-tabs-content' }
+        .tab-pane.fade.show.active{ id: 'packages', role: 'tabpanel', aria: { labelledby: 'packages-tab' } }
+          = render partial: 'project_packages', locals: { project: @project, packages: @packages }
+        .tab-pane.fade{ id: 'inherited-packages', role: 'tabpanel', aria: { labelledby: 'inherited-packages-tab' } }
+          = render partial: 'project_inherited_packages', locals: { ipackages: @ipackages }
```

Comment:
Let's rename `ipackages` to `involved_packages` in the locals. It will be a bit clearer. We should not rename the instance variable `@ipackages` because it would involve also changing the old view (which we don't want to do now).